module-4/metascan/README.md

```diff
@@ -54,7 +49,11 @@ This is the main function that orchestrates the entire process. It parses comman
 ## Usage
 
 1. **Setup Environment**:
-    - Ensure Python 3.x and necessary libraries (`requests`, `beautifulsoup4`, `PyPDF2`) are installed.
+    - Ensure Python 3.12.3 and necessary libraries (`requests 2.31`, `beautifulsoup4 4.12`, `PyPDF2 2.12`) are installed.
```

Comment:
add a link, e.g. `"see [requirements.txt](./path/to/your/requirements/file.txt)"`